worldedit-core/src/main/java/com/sk89q/worldedit/world/NullWorld.java

```diff
@@ -78,7 +78,7 @@ public String getNameUnsafe() {
     //FAWE end
 
     @Override
-    public String getId() {
+    public String id() {
```

Comment:
Thoughts on altering the overriden method?

Reply:
Should be fine as long as `getId()` calls `id()`